packages/web-app-user-management/src/components/Groups/CreateGroupModal.vue

```diff
@@ -9,15 +9,17 @@
     @confirm="$emit('confirm', group)"
   >
     <template #content>
-      <oc-text-input
-        id="create-group-input-display-name"
-        v-model="group.displayName"
-        class="oc-mb-s"
-        :label="$gettext('Group name') + '*'"
-        :error-message="formData.displayName.errorMessage"
-        :fix-message-line="true"
-        @input="validateDisplayName"
-      />
+      <form autocomplete="off">
```

Comment:
I guess the fact that you added a `form` tag instead of adding the `autocomplete="off"` attribute to the `oc-text-input` tag is just in preparation of more input fields coming to the group creating modal at some point in the future, right?

Reply:
correct